### PR TITLE
Monse: New command to list bank accounts

### PR DESCRIPTION
**Pull request by victor-falcon**
## Description

SCR-20230217-um5

image

- New command to list our bank accounts with the current balance
- And update previous transaction commands to show details on side

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/monse/src/bank-accounts.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    </List>
+  );
+}
+
+function getActions(account: BankAccount, refresh: () => void) {
+  return (
+    <ActionPanel title="Actions">
+      <Action.Push
+        shortcut={{ key: "enter", modifiers: [] }}
+        title="Update balance"
```
Comment (pernielsentikaer) [inline extensions/monse/src/bank-accounts.tsx:36]:
```suggestion
        title="Update Balance"
```

File: extensions/monse/src/bank-accounts.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        icon={Icon.Pencil}
+        target={
+          <UpdateBalanceForm
+            account={account}
+            refresh={() => {
+              setTimeout(() => refresh(), 1000);
```
Comment (pernielsentikaer) [inline extensions/monse/src/bank-accounts.tsx:42]: Can you call revalidate from useFetch here?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/monse/src/bank-accounts.tsx (hunk 2 above)
Comment (victor-falcon) [reply extensions/monse/src/bank-accounts.tsx:42]: It's not working without the delay, to be honest don't now why :(

Comment (victor-falcon) [reply extensions/monse/src/bank-accounts.tsx:42]: Well, I think it's because the balance it's updated asynchronously on the server.

Comment (pernielsentikaer) [reply extensions/monse/src/bank-accounts.tsx:42]: You should be able to use [optimisticUpdate](https://developers.raycast.com/utilities/react-hooks/useexec#mutation-and-optimistic-updates) for this 😊

Comment (victor-falcon) [reply extensions/monse/src/bank-accounts.tsx]: I just update this to use optimisticUpdate. Re check the pull request one you can :)

Comment (pernielsentikaer) [reply extensions/monse/src/bank-accounts.tsx]: SO much better and faster 🔥

**Pull request merged**